src/BenchmarkDotNet/Running/BenchmarkRunnerDirty.cs

```diff
@@ -125,5 +177,16 @@ private static Summary[] RunWithExceptionHandling(Func<Summary[]> run)
                 return new[] { Summary.NothingToRun(e.Message, string.Empty, string.Empty) };
             }
         }
+        private static void PrintList(ILogger nonNullLogger, IConfig effectiveConfig, IReadOnlyList<Type> allAvailableTypesWithRunnableBenchmarks, CommandLineOptions options)
```

Comment:
please reuse the method from `BenchmarkSwitcher` and avoid having a copy:

https://github.com/dotnet/BenchmarkDotNet/blob/916dc355d2cd4845923a29afca089650fc6a7573/src/BenchmarkDotNet/Running/BenchmarkSwitcher.cs#L112

You can make the `BenchmarkSwitcher.PrintList` method internal or move this logic to `BenchmarkCasesPrinter`

Reply:
moved ``PrintList() ``` method to ```BenchmarkCasesPrinter```